kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/config/BigQuerySinkTaskConfig.java

```diff
@@ -226,6 +255,22 @@ private void checkSchemaUpdates() {
     return Optional.ofNullable(getList(BIGQUERY_CLUSTERING_FIELD_NAMES_CONFIG));
   }
 
+  /**
+   * Returns the table cache size.
+   * @return Maximum number of tables that can be stored in the cache.
+   */
+  public Optional<Long> getCacheSize() {
+    return Optional.ofNullable(getLong(TASK_CACHE_SIZE_CONFIG));
+  }
+
+  /**
+   * Returns the table cache expiry length.
+   * @return Length of time in hours for which a table is stored in the cache.
+   */
+  public Optional<Long> getCacheExpiry() {
+    return Optional.ofNullable(getLong(TASK_CACHE_EXPIRE_CONFIG));
+  }
+
```

Comment:
Discussed elsewhere, commenting here for the record--don't think we want to add these configurable properties unless we opt to use an expirable cache instead of a simple hash map.